microsoft-edge/webview2/get-started/winforms.md

```diff
@@ -28,19 +28,25 @@ Install the following list of prerequisites before proceeding.
 
 Start with a basic desktop project that contains a single main window.  
 
-1.  In Visual Studio, choose **Windows Forms .NET Framework App** > **Next**.
+1. Open Visual Studio. In the opening panel, choose **Create a new project**.
```

Comment:
```suggestion
1. Open Visual Studio. In the opening panel, select **Create a new project**.
```
Source: public (or internal) version of MWSG, article "Describing interactions with UI", row: "Select" (& row "Choose"). 
 https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui